releasenotes/notes/langchain-lcel-stream-calls-bff85c974a72cceb.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    langchain: Adds support for tracing ``stream`` calls on LCEL chains, chat completion models, or completion models.
+    Due to an upstream issue with the ``langchain`` library itself, streamed responses are not tagged correctly when the `n` parameter on the model is set to greater than 1.
```

```suggestion
    Note that due to an upstream issue with the ``langchain`` library itself, streamed responses will not be tagged correctly based on the choice index when the underlying model is configured to return ``n>1`` choices.
```
Can we also link the associated github issue in this release note entry?